### ISSUE TITLE
Feature: Add link to Backgrounds from character creation screen

### ISSUE THREAD

**Issue by zeg-io** [emfmesquita/beyondhelp#8]
The current character creation - background page doesn't actually display the full text of the background.  It would be great to have a link to the background page.

**Comment by emfmesquita** [comment]
Just a question. The issue is about the character builder screen (the anvil page on character sheet). Or the character sheet itself (the new character sheet).

Because I thought and checked the first one, but it seems all the info was there (at least for acolyte). But if it is for the second one, not sure if this is the right moment to do this issue, since the new character sheet is still changing a bunch since launch.

**Comment by zeg-io** [comment]
Yeah, so most of the info is there, but stuff that has tables (Soldier comes to mind) that has flavor items listed, don't show.

See "Specialty": [https://www.dndbeyond.com/characters/backgrounds/soldier](https://www.dndbeyond.com/characters/backgrounds/soldier)
